### PR TITLE
upgraded dependencies and using the new naming for the deprecated Can…

### PR DESCRIPTION
**Pull request by mihaisoloi**
…celables

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import com.typesafe.sbt.pgp.PgpKeys
 import sbtrelease.ReleaseStateTransformations._
 
-val monixVersion = "2.3.0"
+val monixVersion = "3.0.0-M3"
```
Comment (Avasil) [inline build.sbt]: I think we should upgrade to just ```2.3.3``` and do seperate branch for ```3.0.x```

**Comment by alexandru** [comment]
Thanks Mihai, I've updated the dependencies - Monix is now at 2.3.3. We need a separate branch if we want to provide 3.0.0-x support.

If you're interested, do a PR 😜

**Comment by mihaisoloi** [comment]
will do a separate PR later today for 3.0.0 on a separate branch

File: build.sbt (hunk 1 above)
Comment (mihaisoloi) [reply build.sbt]: agreed, thanks

**Comment by Avasil** [comment]
@mihaisoloi 
Hi, there seems to be growing need for version with `3.0.0` so we'd like to release it in the matter of days. 
I'd love to know if you have time to open this PR soon - if you're busy right now then that's okay just please let me know. :)

**Comment by alexandru** [comment]
Now that we have support for Kafka 1.0 (AFAIK, please correct me if I'm wrong) which was kind of urgent, the sensible thing to do is to ...

1. create a branch called `monix/2.x` from the current master, which will be available going forward for bug fixes
2. update `master` to depend on Monix 3.0

I would release monix-kafka version `1.0.0` with Monix 3.0, but until a stable Monix 3.0 comes out, we need a temporary versioning scheme.

Guess we could do `1.0.0-M3` (where `Mx` follows the Monix version).

What do you say guys?

**Comment by mihaisoloi** [comment]
@alexandru @avasil seems reasonable to me. I created another PR with just the changes to be merged into the `master` branch. As for the `monix/2.x` I hope one of you guys can create it, before merging this https://github.com/monix/monix-kafka/pull/31

**Pull request closed without merging**